### DISCUSSION TITLE
能否制作一个安卓版本的app？

### DISCUSSION THREAD

**Discussion by snakexgc** [XIU2/CloudflareSpeedTest#190] [讨论]
手机端用户不能通过软件获取cf的速度，希望能够制作一个安卓端的app

**Comment by XIU2** [comment]
安卓编程完全是另一个领域，这我无能为力。

目前可以在手机上安装**微型 Linux 虚拟机 APP** 的方式去凑活着用 #61

**Reply by snakexgc** [reply]
好吧，曲线救国！
